modules/swagger-codegen/src/main/resources/JavaPlayFramework/newApiController.mustache

```diff
@@ -0,0 +1,160 @@
+        {{{dataType}}} {{paramName}};
+        {{^required}}
+        if (value{{paramName}} != null) {
+            {{paramName}} = {{>conversionBegin}}value{{paramName}}{{>conversionEnd}};{{/required}}
+        {{#required}}{{paramName}} = {{>conversionBegin}}value{{paramName}}{{>conversionEnd}};{{/required}}
+        {{^required}}
+        } else {
+            {{paramName}} = {{>paramDefaultValue}};
+        }{{/required}}
+        {{/collectionFormat}}
+        {{/queryParams}}
+        {{#formParams}}
+        {{^notFile}}
+        Http.MultipartFormData.FilePart {{paramName}} = request().body().asMultipartFormData().getFile("{{paramName}}");
+        {{#required}}if (({{paramName}} == null || ((File) {{paramName}}.getFile()).length() == 0)) {
+            throw new RuntimeException("File cannot be empty");
+        }
+        {{/required}}
+        {{/notFile}}
+        {{#notFile}}
+        {{#collectionFormat}}
+        //TODO: Maybe implement this in the future if we can support collection in the body params: see bug in swagger-play: https://github.com/swagger-api/swagger-play/issues/130
+        //TODO: Tt seems it is not detected that it's a list based on the collectionFormat field?
+        //WIP when both bugs will be fixed
+        //List<Pair> {{paramName}}Pair = SwaggerUtils.parameterToPairs("{{collectionFormat}}", "{{paramName}}", ((String[]) request().body().asMultipartFormData().asFormUrlEncoded().get("{{paramName}}"))[0]);
+        {{{dataType}}} {{paramName}} = new Array{{{dataType}}}();
+        //for (Pair pair : {{paramName}}Pair) {
+        //    {{paramName}}.add({{>conversionBegin}}pair.getValue(){{>conversionEnd}});
+        //}
+        {{/collectionFormat}}
+        {{^collectionFormat}}
+        String value{{paramName}} = ((String[]) request().body().asMultipartFormData().asFormUrlEncoded().get("{{paramName}}"))[0];
+        {{{dataType}}} {{paramName}};
+        {{^required}}
+        if (value{{paramName}} != null) {
+            {{paramName}} = {{>conversionBegin}}value{{paramName}}{{>conversionEnd}};{{/required}}
+        {{#required}}{{paramName}} = {{>conversionBegin}}value{{paramName}}{{>conversionEnd}};{{/required}}
+        {{^required}}
+        } else {
+            {{paramName}} = {{>paramDefaultValue}};
+        }{{/required}}
+        {{/collectionFormat}}
+        {{/notFile}}
+        {{/formParams}}
+        {{#headerParams}}
+        {{#collectionFormat}}
+        //TODO: Maybe implement this in the future if we can support collection in the body params: see bug in swagger-play: https://github.com/swagger-api/swagger-play/issues/130
+        //TODO: Tt seems it is not detected that it's a list based on the collectionFormat field?
+        //WIP when both bugs will be fixed
+        //List<Pair> {{paramName}}Pair = SwaggerUtils.parameterToPairs("{{collectionFormat}}", "{{paramName}}", request().getHeader("{{baseName}}"));
```

Comment:
My question was about the `"{{paramName}}"` in the SwaggerUtils.parameterToPairs() call.

I have no idea what this pair list is good for, but as this line is commented out, I guess it will not break anything.